spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarTemplateTests.java

```diff
@@ -103,18 +102,28 @@ void sendMessageWithSpecificSchemaTest() throws Exception {
 		}
 	}
 
+	@Test
+	@SuppressWarnings("unchecked")
+	void sendMessageWithEncryptionKeysTest() throws Exception {
```

Comment:
**[NIT]** We can leave off the "Test" suffix as it is only in the parameterized flavors to distinguish the test vs the provider of the arguments to the test ("Test / Provider").

Reply:
done